index.js

```diff
@@ -153,17 +153,17 @@ function setupPath(newPathEntries) {
   core.exportVariable('PATH', [...newPathEntries, ...cleanPath].join(path.delimiter))
 }
 
-function readBundledWithFromGemfileLock() {
-  if (fs.existsSync('Gemfile.lock')) {
-    const contents = fs.readFileSync('Gemfile.lock', 'utf8')
+function readBundledWithFromGemfileLock(path) {
```

Comment:
We might want to rename the function to e.g. `readBundledWithFromGemLockfile`.

Reply:
I think it's OK as-is.